polyfill/test/parse.mjs

```diff
@@ -0,0 +1,43 @@
+#! /usr/bin/env -S node --experimental-modules
+
+// Copyright (C) 2020 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+import Demitasse from '@pipobscure/demitasse';
+const { describe, it, report } = Demitasse;
+
+import Pretty from '@pipobscure/demitasse-pretty';
+const { reporter } = Pretty;
+
+import { strict as assert } from 'assert';
+const { equal, throws } = assert;
+
+import * as Temporal from 'tc39-temporal';
+
+describe('Temporal.parse', () => {
+  it('is a function', () => equal(typeof Temporal.parse, 'function'));
+  it('fails for invalid ISO strings', () => throws(() => Temporal.parse('this is an invalid ISO string'), RangeError));
+  it('does not fail for valid ISO strings', () => assert(() => Temporal.parse('1937-01-01T12:00:27.870+00:20')));
+
+  it('can be used to salvage valid parts of invalid ISO strings', () => {
+    const date = Temporal.Date.from('1922-12-30');
+    const salvagedDate = Temporal.Date.from(Temporal.parse('1922-12-30T99:99:99Z').date);
+    equal(date.toString(), salvagedDate.toString());
```

Comment:
I think it's fine to just put `'1922-12-30'` here instead of constructing `date`, because we just construct it from the string and then call toString on it.

Reply:
Got it.